### ISSUE TITLE
replication finishes but every replica do not join the cluster

### ISSUE THREAD

**Issue by Hero9909** [bitnami/charts#16061]
### Name and Version

bitnami/mariadb-galera

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. `helm install test bitnami/mariadb-galera --set livenessProbe.enabled=false`
2. wait for all nodes to be ready

### What is the expected behavior?

nodes join into one cluster

### What do you see instead?

log states that every node is its own cluster

### Additional information

I've searched through the recent issues with the keywords galera and join or replica, but no update of the repo or other things solved the issues. so I've done this minimal install. using persistant volume (host-path). I verified that selinux is enforcing and file context is valid for the host path storage.

**Comment by Hero9909** [comment]
i've additionally verified via connection on each replica, that each replica is running its own cluster

**Comment by carrodher** [comment]
Are you able to reproduce the issue without setting `livenessProbe.enabled=false`?

**Comment by Hero9909** [comment]
in this case, the nodes are too slow to start and end in an endless chrashloop

**Comment by Hero9909** [comment]
Ok, modified the liveness probe, still same result as with disabled. i did a fresh install, just to make shure no error persists

Still change in behaviour to notice is that as expected not the liveness probe hits very late and does not restarts the replicas.

**Comment by carrodher** [comment]
It seems a very specific use case difficult to reproduce on our side and very tied to your scenario.

For information regarding the application itself, customization of the content within the application, or questions about the use of the technology or infrastructure; we highly recommend checking forums and user guides made available by the project behind the application or the technology.

That said, we will keep this ticket open until the stale bot closes it just in case someone from the community adds some valuable info.

**Comment by Hero9909** [comment]
Let's update these issues, we found the problems and just in case someone stumbles across them again:
When the internal DNS is not working correctly, each replica builds its own cluster. 
for troubleshooting see [kubernetes debugging dns problems](https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/)
Under this symptom, we discovered a misconfiguration in our NSX-T. 

I have checked my own cluster in libvirt and cannot reproduce this problem there. 
I would like to apologize for the inconvenience and trouble I caused. This problem was never caused by this chart.

This could also be a nice check when initializing replicas to ensure a healthy pod.
